aries_vcx/src/messages/proof_presentation/presentation_request.rs

```diff
@@ -12,9 +12,8 @@ pub struct PresentationRequest {
     pub comment: Option<String>,
     #[serde(rename = "request_presentations~attach")]
     pub request_presentations_attach: Attachments,
-    #[serde(skip_serializing_if = "Option::is_none")]
     #[serde(rename = "~thread")]
-    pub thread: Option<Thread>,
+    pub thread: Thread
```

Comment:
Since PresentationRequest is initial message within protocol (typically), does it have `~thread`? Isn't the value of thread determined by message `@id` once the request is received by Prover?

Reply:
Good catch, I missed that. I also think presentation request should not have `~thread`.